### PR TITLE
Fetch metadata and body by UID

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Related to https://github.com/ooni/explorer/issues/595
It's also a step towards removing the report_id/input from many functions leading to a code cleanup.

### CODE REVIEW

File: api/tests/integ/test_integration.py [hunk 1]
```diff
@@ -378,6 +378,67 @@ def test_get_measurement_meta_full(client):
...
+        "probe_asn": 4818,
+        "probe_cc": "MY",
+        "scores": '{"blocking_general":1.0,"blocking_global":0.0,"blocking_country":0.0,"blocking_isp":0.0,"blocking_local":0.0,"analysis":{"blocking_type":"http-failure"}}',
+        "report_id": "20210709T004340Z_webconnectivity_MY_4818_n1_YCM7J9mGcEHds2K3",
+        "test_name": "web_connectivity",
+        "test_start_time": "2021-07-09T00:43:40Z",
+        "category_code": "",
+    }
+
+
```
Comment (bassosimone) [inline api/tests/integ/test_integration.py:419]:
```suggestion
```

The diff seems to add two back-to-back equal tests.

**Review by bassosimone — APPROVED** [review]
LGTM, with a couple of minor comments!

🐳

File: api/tests/integ/test_integration.py [hunk 2]
```diff
@@ -378,6 +378,67 @@ def test_get_measurement_meta_full(client):
...
+def test_get_measurement_meta_by_uid_full(client):
+    uid = "20210709005529.664022_MY_webconnectivity_68e5bea1060d1874"
+    response = api(client, f"measurement_meta?measurement_uid={uid}&full=True")
```
[the remark this answers is not in the document]
Comment (FedericoCeratto) [reply api/tests/integ/test_integration.py:422]: The API is case insensitive as a way to avoid confusion.

**Comment by FedericoCeratto** [comment]
1.0.54~pr656-44 deployed to test

**Review by bassosimone — APPROVED** [review]
🐳

**Comment by FedericoCeratto** [comment]
1.0.54~pr656-49 deployed on FSN

**Pull request merged**